src/index.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable linebreak-style */
+import './styles.css';
+import { scoreList, updateList } from './modules/scorelist';
```

Comment:
Good job working on the project requirement while I was reviewing your work I noticed you disabled eslint, it is against microverse policy to disable linters because linters help us write clean code so it would be nice if you enabled eslint in this file.

Reply:
Thank you for the feedback.